### ISSUE TITLE
Run this code on vck5000 got stuck

### ISSUE THREAD

**Issue by zzy82518996** [FPGAtestbed/vck5000_sum_example#1]
I tried this example on vck5000 but got stuck when running the host code. I am able to generate the libadf.a and xclbin and build the x86 host code. May I ask what is the reason that may cause this issue ?

```
Running sum example on VCK5000 with 32 elements
Load sum.hw.xclbin
Create kernels
Create input and output device buffers
Create runner
Transfer input data to device... Done
```
According to the terminal, it got stuck after transfering input data. The aie graph only set run 1 iteration, I think this simple code should not take that long time right ?

The command I use is: 
```
./host xxx.xclbin 32 
```

**Comment by mesham** [comment]
Are you using our FPGA testbed or your own VCK? On our testbed `module load vitis/2022.1` in a fresh terminal will bring in the correct environment, this is important as some libraries can seem to lock up the vck5000 if they are present when a bitstream is built.

**Comment by zzy82518996** [comment]
Thanks for the reply. 
I am using the vck5000 development card installed on our own server. 
According to your reply, you mean this is becuase of the incorrect xcbin that was build using different environment ? So the host code is correct right ? 
I just wonder which library will lock up the vck5000 ? 
Because when this code stuck, I reset the card and then execute another example from official vitis example which only use pL kernel on vck5000, it works.
I also try to validate the card, it also passes the validation.
Is it necessary to use module load to setup vitis environment ? 
I use CentOS 7 and I source the script files everytime I log into the server. I think I am curious which additional environment variables are needed to make this code available to be deployed on my vck5000 card.

Below is my environment setup:
```
#!/bin/bash
echo "Setting up Vitis 2022.1 ..."
unset LM_LICENSE_FILE
#floating license: departemental server
export XILINXD_LICENSE_FILE=2100@flex.ece.cornell.edu
#Setup path
export VITIS=/work/shared/common/CAD_tool/Xilinx/Vitis-2022/Vitis/2022.1
export HLS_INCLUDE=/work/shared/common/CAD_tool/Xilinx/Vitis-2022/Vitis_HLS/2022.1/include
source $VITIS/settings64.sh
source /opt/xilinx/xrt/setup.sh
```

**Comment by mesham** [comment]
That's just what I found in setting up our VCK5000s, this was into an existing system with numerous bits and pieces and took a bit of experimentation before things would build and run OK. As I say, this code runs fine on our VCK5000, hence me suspecting that this could be the culprit for you. 

It might be worth you clearing out your bash (e.g. ensuring path, ld_library_path etc are all empty) then sourcing the xrt and vitis sh scripts, rebuilding and seeing if that works. For us, I suspected that there was a library or setting somewhere that was interfering with AIE compilation and that was causing the hanging (as like you I could build fine for the PL, but when bringing in the AIE then it caused issues).

Xilinx provides an image processing example for the VCK5000 (downloadable from the early access part of their website) - I suggest trying to build and run that, as if it's an issue with your environment/setup then that would not run either so might help you diagnose the problem to figure out if it's an issue with this code or something a bit wider.
